.yamato/_run-all.yml

```diff
@@ -18,6 +18,17 @@ run_all_tests:
     - .yamato/project-tests.yml#test_{{ project.name }}_{{ editor }}_{{ platform.name }}
 {% endif -%}
 {% endfor -%}
+
+## Test minimal project with different versions of dependencies
+{% if project.name == "minimalproject" -%}
+{% for dependency in dependencies -%}
+{% for depeditor in dependency.test_editors -%}
+{% if depeditor != "trunk" -%}
+    - .yamato/package-tests.yml#test_compatibility_{{ project.name }}_{{ project.packages.first.name }}_with_{{ dependency.name }}@{{ dependency.version }}_{{ depeditor }}_{{ platform.name }}
+{% endif -%}
+{% endfor -%}
+{% endfor -%}
+{% endif -%}
```

Comment:
Not really liking the nested for's everywhere. Should look into changing that in the future.